blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+
+Sometimes in life we don’t realize how one simple action, one simple mistake affects our life until it pushes us to a wall, thats when it hits us hard that we have been heading to wrong direction. Imagine finishing four years in university calling yourself a developer because you knew how to program. Hackathons didn’t pass me by, Ruby meet ups, Android app development classes and tech conferences. I wanted to be every where.
+Not until I finished university and had to put my skills on practice just to realize the only code I could write comfortably was “Hello World” but in different programming languages, that’s when it downed on me that I had been carrying the name “Developer” on my back.
+
+![Sad Truth!](/img/blog/2018/akaringular_languages.jpeg)
+
+
+I had to lay down the load and let it grow in me. So I made a decision, a decision to concentrate on one programming language, putting all efforts on it. Talk of reading books, attending conferences on same, teaching others get started, hanging around people with the same interest as me and making one a mentor.
+
+Thanks to Rails Girls Nairobi started by [StellaMaris Njage](https://github.com/StlMaris123),
+[Ceciliah Mbugua](https://github.com/cesswairimu), [stella ireri](https://github.com/stlireri) and [I](https://github.com/rachaelkiteme) because this shaped my development journey. This is where my Open Source Project contribution journey started, then Rails Girls Summer of Code came in when I was still learning, almost gave up on it because I knew was not yet ready enough. I knew nothing much you know.
+
+>"If we wait for the moment when everything, absolutely everything is ready, we shall never begin” by Ivan Turgenev
+
+The quote!!
+
+Yes, we had to try it out RGSoC internship program 2018, never doubting my abilities anymore and luckily the trial turned to a surprise video, it was a shock at fast, took a minute to read Vaishali Thakkar tough question who’s answer was just a huge Hurrray!!!!!! shout out.We made it!:tada: :tada: :tada: :tada:
+
+Tik Tok, The clock was ticking and July was fast approaching, and only two days to the commencement of the internship my teammate StellaMaris Njage lost her Laptop :sob: :sob:. That was really sad, but this didn’t stop us from getting started. What are friends for? stella ireri helped her with one. Yeah, she got set lucky enough to have our kick off call on time together and our new tech journey with RGSoC began!
+
+
+I love the journey so far, RGSoC has instilled in me coding schedule, making sure that I learn something daily, such volunteering mentors and supervisors to help me when stack and making sure I work with a close partner, someone am used to and always close to. I cant regret being here because am growing drastically each day as a tech lady and career wise. Am ready enough to work with the community till the end of the summer. I can’t wait to see what I will have grown to. Am grateful to RGSoC community so far.
```

Comment:
Bunch of minor issues. Change 'stack->I'm stuck' , 'with a close partner->closely with a partner', 'grown to->grown into'.